src/app/add-data-set-dialog/add-data-set-dialog.component.html

```diff
@@ -0,0 +1,30 @@
+<h2 mat-dialog-title>Add new projects</h2>
```

Comment:
You should explore the package structure in https://github.com/Clean-CaDET/platform-tutor-ui-web/tree/master/src/app and follow it as much as possible. Obviously your modules will be different, and we should discover which modules the front-end app will have as we go along.
An obvious first module is the dataset. We might decide to split it as it grows.

We don't want our components to have the "app" prefix. You can set this in the angular.json https://github.com/Clean-CaDET/platform-explorer-ui-web/blob/362a62a2f8d06fe13d8a578da716ac1e24ce7aa5/angular.json#L15
A good prefix would be "de" (dataset explorer). For all generated components you can either regenerate them or manually change them. @simsimkic pay attention to this comment, as we should ensure that a good structure is set and followed.